.github/workflows/black.yml

```diff
@@ -0,0 +1,10 @@
+name: Lint
+
+on: [push, pull_request]
```

Comment:
This will trigger duplicate check runs.

Reply:
![image](https://user-images.githubusercontent.com/166637/147159349-336faa47-6b62-48a7-8a5e-43833da27dc8.png)

Reply:
This is the recommended config in the black docs: https://black.readthedocs.io/en/stable/integrations/github_actions.html#usage

According to [this community post](https://github.community/t/duplicate-checks-on-push-and-pull-request-simultaneous-event/18012/4), they are actually testing two independent refs:

> The workflow triggered by push and pull_request use different GITHUB_REF. When push to a branch ,the GITHUB_REF of workflow run triggered by push event is the current branch. And workflow run triggered by pull_request event uses refs/pull/:prNumber/merge as its GITHUB_REF. 

If I'm reading that right, `push` will test the head of the current branch, and `pull_request` will test the branch merged into `main`?